shell/platform/windows/flutter_window.cc

```diff
@@ -328,4 +336,25 @@ bool FlutterWindow::NeedsVSync() {
   return true;
 }
 
+void FlutterWindow::OnWindowStateEvent(WindowStateEvent event) {
+  switch (event) {
+    case SHOW:
+      restored_ = true;
+      break;
+    case HIDE:
+      restored_ = focused_ = false;
```

Comment:
Nit: I _thought_ there was something in the google C++ style guide about avoiding chained assignments but I can't for the life of me actually find it. If we do this elsewhere in the code then it's fine to keep it, but if not, we should separate these onto separate lines.

Reply:
Even if it's not explicitly in the style guide, I would consider this less readable (it's easy to miss one of the assignments if you don't read carefully).